flax/core/frozen_dict.py

```diff
@@ -108,7 +108,7 @@ def __hash__(self):
       self._hash = h
     return self._hash
 
-  def copy(self, add_or_replace: Mapping[K, V]) -> 'FrozenDict[K, V]':
+  def copy(self, add_or_replace: 'FrozenDict[K, V]') -> 'FrozenDict[K, V]':
```

Comment:
Side Q: do you remember what python version lets us get rid of the quotes for forward-reference syntax?

Reply:
It might be Python 3.10, but I am having trouble pulling up a Github issue to support this.